compiler/rustc_borrowck/src/type_check/liveness/polonius.rs

```diff
@@ -87,6 +87,8 @@ pub(super) fn populate_access_facts<'a, 'tcx>(
     body: &Body<'tcx>,
     location_table: &LocationTable,
     move_data: &MoveData<'tcx>,
+    //FIXME: this is not mutated, but expected to be modified as
+    // out param, bug?
     dropped_at: &mut Vec<(Local, Location)>,
```

Comment:
This is called here
https://github.com/rust-lang/rust/blob/760e567af5398a0d8c512f904e551e1f38e00d79/compiler/rustc_borrowck/src/type_check/liveness/mod.rs#L55-L59
where drop_used expected to be populated, but it's not.